config/clickhouse/config.xml

```diff
@@ -1,3 +1,16 @@
 <yandex>
     <max_server_memory_usage_to_ram_ratio from_env="MAX_MEMORY_USAGE_RATIO" />
+
+    <zookeeper>
+        <node>
+            <host>f934d2b52573</host>
```

Comment:
Where does that `f934d2b52573` host value come from?

Reply:
Just changed it to "sentry_zookeeper", as the hostname.